src/annotator/range-util.js

```diff
@@ -150,8 +152,8 @@ export function itemsForRange(range, itemForNode) {
       checkedNodes.add(current);
 
       const item = itemForNode(current);
-      if (item) {
```

Comment:
The previous behaviour is slightly than the new one. Before it was a 'truthy' test, which will filter `null|undefined|0|''|false`, while the new behaviour filters `null|undefined` (`NonNullable`).

The only place where this functionality is used is `Guest` and there, it doesn't make a difference.